django/db/backends/sqlite3/creation.py

```diff
@@ -106,3 +122,29 @@ def test_db_signature(self):
         else:
             sig.append(test_database_name)
         return tuple(sig)
+
+    def setup_worker_connection(self, _worker_id):
+        alias = self.connection.alias
+        worker_db = f"file:memorydb_{alias}_{_worker_id}?mode=memory&cache=shared"
+
+        start_method = multiprocessing.get_start_method()
+        if start_method == "fork":
+            source_db = sqlite3.connect(
+                f"file:memorydb_{alias}?mode=memory&cache=shared", uri=True
+            )
+        elif start_method == "spawn":
+            source_db = sqlite3.connect(
+                "file:%s_%s.sqlite3" % (alias, _worker_id), uri=True
```

Comment:
Everywhere else uses an f-string, not sure why this doesn't:
```suggestion
                f"file:{alias}_{_worker_id}.sqlite3", uri=True
```

Reply:
I'm slightly confused that were taking `source_db` from disk here with the same `_worker_id` value as the `worker_db` target. Maybe I'm missing something, but I thought we're trying to make copies for different processes, so the source and target cannot share the same `_worker_id`? 😕

Reply:
Here I think we're opening an already copied database from a file back into memory, so it's ok for them to have the same name.